File: ws/quiroz/U1/WS12ProjectStructure/ChickenFarmSystem.java
package ec.edu.espe.farmSimulator.model.view;

import ec.edu.espe.farmsimulator.model.model.Egg;
import ec.edu.espe.farmsimulator.model.model.Poop;

/**
 *
 * @author Maria Quiroz
 */

public class ChickenFarmSystem {
    public static void main(String args[]){
        
        System.out.println("This is a Chicken Farm Simulator");
        
        //Declare
        Poop poop;
        Egg egg;
        Egg eggTwo;
        Egg eggThree;
        int numberOfTires;
        String name;
        
        //Initialize
        poop = new Poop();
        egg = new Egg();
        eggTwo = new Egg();
        eggThree = new Egg();
        numberOfTires=5;
        name= "Maria Quiroz";
        
        //Methods
        System.out.println("The number of tires is --->" + numberOfTires);
        System.out.println("Chicken poop is -->" + poop);
        System.out.println("My name is  --> "+name);
        egg.setId(1);
        System.out.println("Chicken egg is -->" + egg);
        System.out.println("Chicken egg id is --> " + egg.getId());
        egg.setId(2);
        System.out.println("Chicken egg is -->" + eggTwo);
        System.out.println("Chicken egg id is --> " + egg.getId());
        egg.setId(3);
        System.out.println("Chicken egg is -->" + eggThree);
        System.out.println("Chicken egg id is --> " + egg.getId());
    }
}

File: ws/paillacho/U1/WS12ProjectStructure/ChickenFarmSystem.py
#ChickenFarmSystem

class Poop:
    def __str__(self):
        return "This is poop from a chicken."


class Egg:
    def __init__(self):
        self._id = 0

    def set_id(self, egg_id):
        self._id = egg_id

    def get_id(self):
        return self._id

    def __str__(self):
        return f"Egg{{id={self._id}}}"


def main():
    print("This is a Chicken Farm Simulator")

    # Declarar e inicializar
    poop = Poop()
    egg = Egg()
    egg_two = Egg()
    egg_three = Egg()
    number_of_tires = 5
    name = "Maria Quiroz"

    # Métodos
    print("The number of tires is --->", number_of_tires)
    print("Chicken poop is -->", poop)
    print("My name is -->", name)

    egg.set_id(1)
    print("Chicken egg is -->", egg)
    print("Chicken egg id is -->", egg.get_id())

    egg_two.set_id(2)
    print("Chicken egg is -->", egg_two)
    print("Chicken egg id is -->", egg_two.get_id())

    egg_three.set_id(3)
    print("Chicken egg is -->", egg_three)
    print("Chicken egg id is -->", egg_three.get_id())


if __name__ == "__main__":
    main()
